state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -2266,4 +2315,40 @@ pub(crate) mod utilities {
 			.unwrap_or(u128::MAX),
 		}
 	}
+
+	// Note:
```

Comment:
Missing note :)

Reply:
Don't remember what I wanted to say, so I just deleted it now :)